prow/jobs/test-infra/prow-periodics.yaml

```diff
@@ -283,6 +283,6 @@ periodics: # runs on schedule
             command:
               - "generic-autobumper"
             args:
-              - "--config=configs/autobump-config/test-infra-autobump-config.yaml"
+              - "--config=prow/autobump-config/test-infra-autobump-config.yaml"
```

Comment:
is this path correct after refactor of test infra?  I think it should be now here: https://github.com/kyma-project/test-infra/tree/main/configs/autobump-config

Reply:
corrected